lib/fiware-iotagent-lib.js

```diff
@@ -114,20 +113,6 @@ function doActivate(newConfig, callback) {
 
     config.setConfig(newConfig);
 
-    logger.getContext = function domainContext() {
```

Comment:
Does this removal mean that this function is not actually used in this file? I mean, you have done some cleanup here.

Reply:
Ah no... it's just a move. I see now arround L241.

NTC

Reply:
Not really, I need to have configured the logger in the startServer() function to write properly the logs, therefore I translate this functionality at the beginning of that function.

Reply:
But the doActivate() also need to set the logger.getContext function... isn't it? I mean, the addition to startServer() is fine but not sure about the removal in doActivate()

(Not fully sure... the logger library is tricky and I'm not fully sure of the details)

Reply:
When doActivate() is called the logger has been configured accordingly in the startServer function.

Reply:
Maybe not so evident... I'd suggest to add a comment about it in the JavaDoc for doActivate() (around L75) to make it clearer. Something like:

> Note that this function is called after startServer() so the logger getConext() function has been properly initialized

Reply:
Fixed in 547946e